simpleregex/func.py

```diff
@@ -61,11 +67,24 @@ def times(what: RegEx, min: int, max: int = None):
     Repeate a pattern between {min} and {max} times.
     """
     what = ensure_regex(what)
-    return what + (f"{{{min},}}" if max is None else f"{{{min},{max}}}")
```

Comment:
in regex "{N}" means N times (not more, not less). "{N,}" means at least N times. This should not be changed.